Sources/Sentry/include/SentrySwizzle.h

```diff
@@ -152,6 +152,10 @@ typedef void (*SentrySwizzleOriginalIMP)(void /* id, SEL, ... */);
 /// The selector of the swizzled method.
 @property (nonatomic, readonly) SEL selector;
 
+#if TEST
```

Comment:
`h`: I think it would be acceptable to add this property in every build configuration. If the original method is not called something is clearly wrong and the SDK should crash.